doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,124 @@
+The local AHS device
+====================
+
+The local analog hamiltonian simulation (AHS) device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS simulator <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
+
+The local AHS device is not gate-based. Instead, it is compatible with the `ParametrizedEvolution <https://docs.pennylane.ai/en/stable/code/api/pennylane.pulse.ParametrizedEvolution.html>`_
+operator from `pulse programming <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_ in PennyLane.
+
+Note that pulse programming in PennyLane requires the module ``jax``. You can install jax via: pip install jax==0.4.3 jaxlib==0.4.3
+
+To instantiate the local Braket simulator, simply use:
+
+.. code-block:: python
+
+    import pennylane as qml
+    device_local = qml.device("braket.local.ahs", wires=2)
+
+This device can be used with a QNode within PennyLane. It accepts circuits with a single `ParametrizedEvolution <https://docs.pennylane.ai/en/stable/code/api/pennylane.pulse.ParametrizedEvolution.html>`_
+operator based on a `ParametrizedHamiltonian <https://docs.pennylane.ai/en/stable/code/api/pennylane.pulse.ParametrizedHamiltonian.html>`_ compatible with the simulated hardware.
+More information about creating PennyLane operators for AHS can be found in the `PennyLane docs <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_.
+
+.. note::
+    It is important to keep track of units when specifying electromagnetic pulses for hardware control.
+    The frequency and amplitude provided in PennyLane will be multiplied to a factor of 1e6 when translating
+    to hardware (converted from :math:`2 \pi` MHz to rad/s), while time will be divided by 1e6 (converted from
```

Comment:
Kind of. Currently the plan is to ask PL users to specify in units of $2 \pi$ MHz. But also the plan is to settle this for sure Wednesday morning when Korbinian is back.

Reply:
The idea is to have as little implicitly assumed factors as possible, such that the rule "what the users puts, is also what the program uses" holds. 

Currently, the only caveat is that we still multiply/divide by `1e6` on the device. This is because in numerical simulation these factors cancel out so it makes sense to not put them explicitly, but on the real device it is expecting values in the actual physical order of magnitude.

Alternative is to not have this multiplication and tell users that they need to specify this when running on real hardware. It would make the user more autonomous, but the downside is that you have to manually do this step when going from simulation to real device.

Reply:
Settled in meeting today - we are expecting standard frequency in PL and converting to angular frequency for the user. We are also converting orders of magnitude so that the order of magnitude expected by the PL ODE will be converted to the correct units for users, rather than asking users to rescale all their parameters to go from PL to Braket.